### PR TITLE
Add reshape in front of a fully connected node for using bf input

### PR DESCRIPTION
**Pull request by byungilm**
Signed-off-by: Min, Byungil <byungil.min@intel.com>

### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp [hunk 1]
```diff
@@ -575,14 +576,32 @@ void reorder_inputs::run(program& p, layout_optimizer& lo, reorder_factory& rf)
         }
     };
 
+    const auto reorder_input_fully_connected = [&p, &lo, &rf](typed_program_node<fully_connected>& fc_node) {
+        auto& weights = fc_node.weights();
+        auto& input = fc_node.input();
+        auto weights_layout = weights.get_output_layout();
+        auto input_layout = input.get_output_layout();
+        // Change input data of fully-connected node from bx to bf
+        if (format::is_simple_data_format(weights_layout.format) && format::is_simple_data_format(input_layout.format) && weights.is_constant() &&
```
Comment (isanghao) [inline inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp:585]: Is it necessary to check `weights_layout`?

Comment (byungilm) [reply inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp:585]: Removed

File: src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp [hunk 2]
```diff
@@ -575,14 +576,32 @@ void reorder_inputs::run(program& p, layout_optimizer& lo, reorder_factory& rf)
         }
     };
 
+    const auto reorder_input_fully_connected = [&p, &lo, &rf](typed_program_node<fully_connected>& fc_node) {
+        auto& weights = fc_node.weights();
+        auto& input = fc_node.input();
+        auto weights_layout = weights.get_output_layout();
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp:582]: this is not being used, is it?

Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp:582]: (saw above comment after I left this.. seems that the weights_layout is left after removal of the above check :)

**Review by isanghao — COMMENTED** [review]
Looks good, please check the impact on networks.

**Review by isanghao — APPROVED** [review]

**Pull request merged**